src/__tests__/__snapshots__/InputField.js.snap

```diff
@@ -316,7 +316,7 @@ exports[`InputField it renders a with a both icons 1`] = `
 .c1 {
   margin-left: 8px;
   margin-right: -32px;
-  margin-top: 8px;
+  margin-top: 2px;
```

Comment:
You can see in the snapshot that it's now the correct value. It was supposed to be 2px for margin top but would get 8px because space[index=2] = 8